detections/network/detect_network_traffic_from_inactive_accounts.yml

```diff
@@ -0,0 +1,94 @@
+name: Detect Network Traffic From Inactive Accounts
+id: e08aa2f6-7d90-4ab8-af11-da4df38bb7ff
+version: 1
+date: '2024-10-14'
+author: Zaki Zarkasih Al Mustafa
+data_sources:
+- Windows Event Log Security 4624
+- Windows Event Log Security 4625
+type: Anomaly
+status: production
+description: This detection identifies users who have been inactive for more than 30 days and suddenly have activity based on network traffic logs.
```

Comment:
Can we improve the detection description to better explain how this search logic works?